fe/fe-core/src/main/java/com/starrocks/alter/LakeTableRollupBuilder.java

```diff
@@ -74,7 +74,7 @@ public AlterJobV2 build() throws StarRocksException {
                 long physicalPartitionId = physicalPartition.getId();
                 // create shard group
                 long shardGroupId = GlobalStateMgr.getCurrentState().getStarOSAgent().
-                        createShardGroup(dbId, olapTable.getId(), partitionId, rollupIndexId);
+                        createShardGroup(dbId, olapTable.getId(), physicalPartitionId, rollupIndexId);
```

Comment:
also with this change, we need to keep in mind that in future if anyone want to use the "partitionId" attached to the shardGroup property, it can be `partitionId` or `physicalPartitionId` based on when the shardGroup is created. May need to record this info somewhere.

Reply:
That's what I am worried about. Need to put notes somewhere to reminder the "future" developer who will make the change, otherwise who knows there is an issue here due to historic change of the tablet path.